### PR TITLE
#105 cannot create new goal

### PR DESCRIPTION
**Pull request by denis1stomin**
Fixed the issue #105. Also added multiline support only for KR and its description.
There is a odd behavior when replacing line breakers with <br> it adds some extra tab symbols.
Can't understand why, will fix it later as a minor issue.

### CODE REVIEW

File: webui/src/components/InputForm.vue [hunk 1]
```diff
@@ -32,11 +45,15 @@
...
                 }
             },
 
             onEnter(text) {
+                this.saveContent(text);
+            },
+
+            saveContent(text) {
                 if(!this.readonly && (text || this.acceptEmpty)) {
                     this.action(text);
```
Comment (denis1stomin) [inline webui/src/components/InputForm.vue]: @skalitin do you remember what is `this.action`?

Comment (denis1stomin) [reply webui/src/components/InputForm.vue]: ah I've found it. Will take a look tomorrow

**Comment by denis1stomin** [comment]
Fixed blur-issue
And trying to make it really multiline. Works not the best way but can fix it later

File: webui/src/components/KeyResult.vue [hunk 2]
```diff
@@ -4,6 +4,7 @@
                     autosave="true"
                     acceptEmpty="true"
                     :readonly="readonly"
+                    :multiline="true"
```
Comment (skalitin) [inline webui/src/components/KeyResult.vue]: As far as I understood it is a static property, so `multiline="true"` is enough (no colon required).

**Review by skalitin — APPROVED** [review]
Looks good! Really critical issue.
Would be good to support Shift-Enter for new line without saving O/KR. But some time later )

File: webui/src/components/KeyResult.vue (hunk 2 above)
Comment (denis1stomin) [reply webui/src/components/KeyResult.vue]: I didn't know the difference. Will fix

**Pull request merged**